### PR TITLE
fix: Removed CR on variants flag (it's GA)

### PR DESCRIPTION
**Pull request by chriswk**
This PR removes the usage of crOnVariants flag, but keeps the behaviour, so CR are now enabled on variants.

### CODE REVIEW

**Review by nunogois — APPROVED** [review]
LGTM 🙂

**Review by nunogois — CHANGES_REQUESTED** [review]
Don't forget about frontend usage of the flag: 

https://github.com/Unleash/unleash/blob/1272a110b0c717de117f2d6cac1f138488c8175e/frontend/src/interfaces/uiConfig.ts#L44

https://github.com/Unleash/unleash/blob/1272a110b0c717de117f2d6cac1f138488c8175e/frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/FeatureEnvironmentVariants.tsx#L73

https://github.com/Unleash/unleash/blob/1272a110b0c717de117f2d6cac1f138488c8175e/frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/EnvironmentVariantsModal.tsx#L274

**Comment by chriswk** [comment]
Thanks, good point. Removed

File: frontend/src/interfaces/uiConfig.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export interface IFlags {
     messageBanner?: boolean;
     featuresExportImport?: boolean;
     caseInsensitiveInOperators?: boolean;
-    crOnVariants?: boolean;
+    /api/admin/projects/default/features/test'iants?: boolean;
```
Comment (nunogois) [inline frontend/src/interfaces/uiConfig.ts:44]: Copy-paste mistake?

```suggestion

```

**Pull request merged**